ec2utils/ec2metadata/MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include LICENSE
+include Makefile
+include python3-ec2metadata.spec
```

Comment:
Lest keep the spec file out of the source distribution as it is not universally applicable. Those interested in packaging the code will need to create their own packaging directive. If they want to use ours as a guideline it is reasonable to assume that they will find this GitHub project

Reply:
Will update in all PR's and rebase for clean commit history.